common/environments/python-sdk-v2/environment.yaml

```
@@ -0,0 +1,9 @@
+image:
+  name: python-sdk-v2
```

```suggestion
  name: azureml/curated/python-sdk-v2
```

This file will also need a `publish` section. See https://github.com/Azure/azureml-assets/wiki/Environments#environmentyaml

And the mcr configuration will also need to be added to the mcr repo. See https://github.com/microsoft/mcr/blob/main/teams/azureml/base/curated-public.yml

---

Done

---

Created a MCR PR here: https://github.com/microsoft/mcr/pull/2418